xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -82,12 +85,22 @@ private LdsUpdate(
           ? null : Collections.unmodifiableList(new ArrayList<>(virtualHosts));
       this.hasFaultInjection = hasFaultInjection;
       this.httpFault = httpFault;
+      this.listener = null;
+    }
+
+    LdsUpdate(Listener listener) {
+      this.listener = listener;
+      this.httpMaxStreamDurationNano = 0L;
```

Comment:
This is redundant. As well as for `hasFaultInjection`, but that would be deleted by #7932...

Reply:
But for now I need to initialize `final` fields with some value and I would like to get this PR in instead of depending on another PR